### DISCUSSION TITLE
Start menu tiles are not clickable when recommended section is disabled

### DISCUSSION THREAD

**Discussion by rouninhp** [valinet/ExplorerPatcher#2184] [Q&A]
Whenever I press start, I can't click on any of the tiles if "recommended section" is disabled. I have to search for them and hit enter to open instead. It usually works well when I turn off my laptop screen to use only an external monitor. However, it does not work on the laptop itself. Any ideas how to make items clickable with "recommended section" disabled?

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.

**Reply by rouninhp** [reply]
Windows 11 version 22H2, build 22621.675.
Explorer patcher version 22621.2361.58.1.
[ExplorerPatcher_22621.2361.58.1.zip - Settings](https://github.com/valinet/ExplorerPatcher/files/12730596/ExplorerPatcher_22621.2361.58.1.zip).

**Comment by pyrates999** [comment]
You're on an old update: https://support.microsoft.com/en-au/topic/october-18-2022-kb5019509-os-build-22621-675-out-of-band-4fdf3458-148a-46bd-b163-ecb93c282726

Please update to the current update: https://support.microsoft.com/en-us/topic/september-12-2023-kb5030219-os-build-22621-2283-8b9ba74f-fdde-4c75-b049-77cf214f4a66
